### PR TITLE
Added next and prev Buttons

### PR DESCRIPTION
**Pull request by f7deleon**
Added next and previous buttons and new question were introduced

### Related Trello cards:
https://trello.com/c/LfjGEJc4

### CODE REVIEW

File: app/src/main/java/com/example/felipe/geoquiz/Question.kt [hunk 1]
```diff
@@ -1,14 +1,5 @@
...
-private val textResId:Int
-private val answerTrue:Boolean
+data class Question(var textResId: Int, var isTrue: Boolean) {
 
-constructor( textResId:Int,  answerTrue:Boolean)
-
-init {
-    this.textResId = textResId
-    this.answerTrue = answerTrue
 }
```
Comment (mirland) [inline app/src/main/java/com/example/felipe/geoquiz/Question.kt]: not needed

File: app/src/main/java/com/example/felipe/geoquiz/QuizActivity.kt [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+import android.support.v7.app.AppCompatActivity
+import android.os.Bundle
+import android.view.Gravity
+import android.widget.Toast
+import kotlinx.android.synthetic.main.activity_quiz.*
+
+
+class QuizActivity : AppCompatActivity() {
+    private val questionBank = arrayOf(
```
Comment (mirland) [inline app/src/main/java/com/example/felipe/geoquiz/QuizActivity.kt]: move it to another class, maybe a controller

File: app/src/main/java/com/example/felipe/geoquiz/QuizActivity.kt [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            false -> R.string.incorrect
+        }
+        with(Toast.makeText(this, msg, Toast.LENGTH_SHORT)) {
+            setGravity(Gravity.TOP, 0, 200)
+            show()
+        }
+    }
+
+}
+
```
Comment (mirland) [inline app/src/main/java/com/example/felipe/geoquiz/QuizActivity.kt]: rm * 2

File: app/src/main/java/com/example/felipe/geoquiz/QuizActivity.kt [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+                false -> questionBank.size - 1
+            }
+            txtQuestion.setText(questionBank.get(index).textResId)
+        }
+
+        btnTrue.setOnClickListener { printResult(true) }
+        btnFalse.setOnClickListener { printResult(false) }
+    }
+
+    private fun printResult(message: Boolean) {
```
Comment (mirland) [inline app/src/main/java/com/example/felipe/geoquiz/QuizActivity.kt]: what do you think splitting this method in two: `checkAnswerResult` and `showAnswerResult`? The first one should call the controller

Comment (mirland) [reply app/src/main/java/com/example/felipe/geoquiz/QuizActivity.kt]: The param name is not good

File: app/src/main/java/com/example/felipe/geoquiz/QuizActivity.kt [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        val msg = when (message == questionBank.get(index).isTrue) {
+            true -> R.string.correct
+            false -> R.string.incorrect
+        }
+        with(Toast.makeText(this, msg, Toast.LENGTH_SHORT)) {
+            setGravity(Gravity.TOP, 0, 200)
+            show()
+        }
+    }
+
```
Comment (mirland) [inline app/src/main/java/com/example/felipe/geoquiz/QuizActivity.kt]: rm

**Review by mirland — CHANGES_REQUESTED** [review]

**Review by FerrariBruno — APPROVED** [review]

File: app/src/main/java/com/example/felipe/geoquiz/QuizController.kt [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+            Question(R.string.question_oceans, true),
+            Question(R.string.question_mideast, false),
+            Question(R.string.question_americas, false),
+            Question(R.string.question_africa, true),
+            Question(R.string.question_asia, true)
+    )
+
+    public fun checkAnswerResult(message: Boolean, index: Int) = message == questionBank.get(index).isTrue
+
+    public fun getQuestion(index: Int) = questionBank.get(index = index).textResId
```
Comment (mirland) [inline app/src/main/java/com/example/felipe/geoquiz/QuizController.kt:15]: the controller should not return the `reid`

**Pull request merged**